### PR TITLE
Add NSXLoadBalancerMonitor to block list

### PR DESCRIPTION
**Pull request by xinyanw409**
**What this PR does / why we need it**:
This pr fix the issue that snapshots being created for kind "NSXLoadBalancerMonitor" is stuck in the "New" phase forever. 
We already skip the CRD with full name "nsxlbmonitors.vmware.com", while "NSXLoadBalancerMonitor" is an accepted name and we don't include it in the block list.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
NOTE: In new vSphere version, contentsourcebindings.vmoperator.vmware.com is new CRD and we don't add it to our block list.

Tested on supervisor cluster.
**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/514/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/515/

**Comment by xing-yang** [comment]
In addition to NSXLoadBalancerMonitor, please take a look of the uploaded file CRDs.rtf and add the additional CRDs to the skip list.

**Comment by xinyanw409** [comment]
> In addition to NSXLoadBalancerMonitor, please take a look of the uploaded file CRDs.rtf and add the additional CRDs to the skip list.

Add new crds in block list.

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -236,6 +237,31 @@ var ResourcesToBlock = map[string]bool{
...
+	"persistenceinstanceinfoes.psp.wcp.vmware.com":            true,
+	"resourcecheckreports.psp.wcp.vmware.com":                 true,
+	"resourcechecks.psp.wcp.vmware.com":                       true,
+	"storagepolicies.psp.wcp.vmware.com":                      true,
+	"tanzukubernetesaddons.run.tanzu.vmware.com":              true,
+	"tcproutes.networking.x-k8s.io":                           true,
+	"virtualmachineclassbindings.vmoperator.vmware.com":       true,
+	"vmxnet3networkinterfaces.netoperator.vmware.com":         true,
+	"vspheredistributednetworks.netoperator.vmware.com":       true,
+	"wcpnamespaces.appplatform.wcp.vmware.com":                true,
```
Comment (xing-yang) [inline pkg/constants/constants.go:264]: Can you sort this in alphabetic order?

Comment (xinyanw409) [reply pkg/constants/constants.go:262]: Sorted in alphabetic order.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/516/

**Comment by xing-yang** [comment]
As discussed, please add "contentsourcebindings.vmoperator.vmware.com".  After that this looks good.

**Review by xing-yang — APPROVED** [review]
LGTM

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/517/

**Comment by xing-yang** [comment]
@xinyanw409 I don't see ""contentsourcebindings.vmoperator.vmware.com" being added to this PR.  Can you add it?

**Pull request merged**